cli/smartcontract/testdata/notifications/rpcbindings_guessed.out

```diff
@@ -103,6 +103,11 @@ type SomeArrayEvent struct {
 	A [][]*big.Int
 }
 
+// SomeUnexportedFieldEvent represents "SomeUnexportedField" event emitted by the contract.
+type SomeUnexportedFieldEvent struct {
+	S *Unnamed
```

Comment:
We already have another `Unnamed` structure in this package nested from other event. This bug will be fixed in a separate issue.

Reply:
Ref. https://github.com/nspcc-dev/neo-go/issues/3088. Let it be "as is" in this PR.